tsunami/1d/pwlinear1/setrun.py

```diff
@@ -0,0 +1,263 @@
+"""
+Module to set up run time parameters for Clawpack -- classic code.
+
+The values set in the function setrun are then written out to data files
+that will be read in by the Fortran code.
+
+"""
+
+import os
+import numpy as np
+
+import clawpack.geoclaw.shallow_1d.data as geoclaw1d
```

Comment:
Getting an error that this shallow_1d.data is not found... is there a branch of geoclaw that is needed for this?